test/integration/targets/ansible-galaxy-collection/vars/main.yml

```diff
@@ -4,13 +4,12 @@ gpg_homedir: "{{ galaxy_dir }}/gpg"
 
 offline_server: https://test-hub.demolab.local/api/galaxy/content/api/
 
+# Test oldest and most recently supported, and versions with notable changes
 supported_resolvelib_versions:
-  - "0.5.3"  # Oldest supported
-  - "0.6.0"
-  - "0.7.0"
-  - "0.8.0"
-  - "0.9.0"
-  - "1.0.1"
+  - "0.5.3"  # test CollectionDependencyProvider050
+  - "0.6.0"  # test CollectionDependencyProvider060
+  - "0.7.0"  # test CollectionDependencyProvider070
+  - "1.1.0"  # test CollectionDependencyProvider080
```

Comment:
If we end up using additional resolvelib features, we may need to add some of the other versions back, but with the features we use now, it's redundant to test 0.8.0, 0.9.0, 1.0.1, AND 1.1.0 since nothing we use has had noteworthy changes since 0.8.0.